### DISCUSSION TITLE
T4 template generate language resources error at System.Runtime

### DISCUSSION THREAD

**Discussion by my522cn** [HandyOrg/HandyControl#1091] [Q&A]
Hello
I write a T4 template as [https://hosseini.ninja/handycontrol/langs/]( https://hosseini.ninja/handycontrol/langs/)
Unfortunately, the it can run, explain that can not find System.Runtime version 6.0.0.
So I tried [https://stackoverflow.com/questions/51550265/t4-template-could-not-load-file-or-assembly-system-runtime-version-4-2-0-0](https://stackoverflow.com/questions/51550265/t4-template-could-not-load-file-or-assembly-system-runtime-version-4-2-0-0 )
But still not work.

Environment：
VS2022 preview
.Net 6.0

**Comment by ghost1372** [comment]
I fixed some issues now you can copy full t4 codes (Previously, some codes were displayed incompletely)
however I do not know if T4 is compatible with .NET 6 or not

**Reply by my522cn** [reply]
it's still not work 😂. I don't know whether it is a bug from .net6
![Snipaste_2021-12-28_14-57-32](https://user-images.githubusercontent.com/11346219/147537837-968aeec9-9dc7-4f3e-a2d6-005f50a6df51.png)
